src/notification/notification-drawer.component.js

```diff
@@ -51,6 +51,12 @@ angular.module('patternfly.notification').component('pfNotificationDrawer', {
       });
     };
 
+    var updateAccordionSizing = function () {
+      $timeout(function () {
```

Comment:
Any concerns about performance here?  if `$onChanges` is triggered frequently, do you want to be triggering resize often, or would a `debounce`/`throttle` be in order?  I'm also curious about `100`, vs (`0` or any other small #) if its arbitrary or standardized across patternfly angular for this kind of thing.